### ISSUE TITLE
JPEG Support?

### ISSUE THREAD

**Issue by daverickdunn** [quodlibet/mutagen#298]
Hi there,

```
import mutagen
mutagen_file = mutagen.File("LP Melodies From Mars (unreleased) CD Cover.jpg")
print(mutagen_file.info.bitrate)
print(mutagen_file.info.length)
```
Results in:

```
5905670160
1485.051
```

Is this expected behaviour?

**Comment by daverickdunn** [comment]
Okay, it appears to be just that file, maybe it's corrupt, maybe it's part of some old malware. I'm still confused as to how mutagen reads files, doe it load all files then look for metadata, as opposed to checking extensions?

I ran pprint on it and got more info:
```
ASF (Windows Media Audio 9 Standard) 5905670160 bps, 360 Hz, 0 channels, 1485.05 seconds (audio/x-ms-wma)
WMFSDKVersion=9.00.00.2980
WMFSDKNeeded=0.0.0.0000
IsVBR=False
IsVBR=False
DeviceConformanceTemplate=L1
IsVBR=False
DeviceConformanceTemplate=MP@ML
```

I've attached it if you want to investigate further. I had to rename from jpg to txt to make it downloadable, otherwise github tries to just display it.

[LP Melodies From Mars (unreleased) CD Cover.txt](https://github.com/quodlibet/mutagen/files/894879/LP.Melodies.From.Mars.unreleased.CD.Cover.txt)

I'm satisfied enough to close this...
